webpack.config.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
     filename: 'moment-range.js',
     library: 'moment-range',
     libraryTarget: 'umd',
-    path: './dist/',
+    path: __dirname + '/dist/',
```

Comment:
Please remove, I've seen there's some issues relating to this so we'll address it elsewhere 😄